### ISSUE TITLE
libupnp does not link on UWP/32bit

### ISSUE THREAD

**Issue by hpvb** [godotengine/godot#21546]
**Godot version:**
Godot 6fcc20ec5116a77c2328f4406a258527cc1aad5e

**OS/device including version:**
UWP

**Issue description:**
Cannot link upnp on 32bit UWP platforms

**Steps to reproduce:**
Build 32bit UWP binaries

```
modules.uwp.opt.32.x86.lib(lws-plat-win.uwp.opt.32.x86.obj) : error LNK2001: unresolved external symbol _CreateFileW
modules.uwp.opt.32.x86.lib(lws-plat-win.uwp.opt.32.x86.obj) : error LNK2001: unresolved external symbol _GetFileSize
```

Note: For the current a1 build upnp is disabled for uwp.

**Comment by akien-mga** [comment]
CC @vnen @mhilbrunner

**Comment by mhilbrunner** [comment]
See also: https://github.com/godotengine/godot/issues/20672

**Comment by Faless** [comment]
Fixed in master via #22044 . Patch is proposed upstream.
